lib/hardware/modules/iob_int_sqrt/iob_int_sqrt.py

```diff
@@ -3,7 +3,237 @@ def setup(py_params_dict):
         "original_name": "iob_int_sqrt",
         "name": "iob_int_sqrt",
         "version": "0.1",
-        "generate_hw": False,
+        "confs": [
+            {
+                "name": "DATA_W",
+                "type": "P",
+                "val": "32",
+                "min": "NA",
+                "max": "NA",
+                "descr": "Data bus width",
+            },
+            {
+                "name": "FRACTIONAL_W",
+                "type": "P",
+                "val": 0,
+                "min": "NA",
+                "max": "NA",
+                "descr": "Reset value.",
+            },
+            {
+                "name": "REAL_W",
+                "type": "P",
+                "val": "DATA_W - FRACTIONAL_W",
+                "min": "NA",
+                "max": "NA",
+                "descr": "Reset value.",
+            },
+            {
+                "name": "SIZE_W",
+                "type": "P",
+                "val": "(REAL_W / 2) + FRACTIONAL_W",
+                "min": "NA",
+                "max": "NA",
+                "descr": "Reset value.",
+            },
+            {
+                "name": "END_COUNT",
+                "type": "F",
+                "val": "(DATA_W + FRACTIONAL_W) >> 1",
+                "min": "NA",
+                "max": "NA",
+                "descr": "Reset value.",
+            },
+            {
+                "name": "COUNT_W",
+                "type": "F",
+                "val": "$clog2(END_COUNT)",
+                "min": "NA",
+                "max": "NA",
+                "descr": "Reset value.",
+            },
+        ],
+        "ports": [
+            {
+                "name": "clk_i",
+                "descr": "Input port",
+                "signals": [
+                    {
+                        "name": "clk",
+                        "width": 1,
+                        "direction": "input",
+                    },
+                ],
+            },
+            {
+                "name": "rst_i",
+                "descr": "Input port",
+                "signals": [
+                    {
+                        "name": "rst",
+                        "width": 1,
+                        "direction": "input",
+                    },
+                ],
+            },
```

Comment:
The `clk_rst` interface is standard and can be auto-generated with `if_gen.py`. No need to specify these ports individually.
```suggestion
            {
                "name": "clk_rst",
                "interface": {
                    "type": "clk_rst",
                    "subtype": "slave",
                },
                "descr": "Clock and reset",
            },

```